pubtools/pulplib/_impl/model/repository/file.py

```diff
@@ -14,6 +14,21 @@
 LOG = logging.getLogger("pubtools.pulplib")
 
 
+@attr.s(kw_only=True, frozen=True)
+class FileSyncOptions(SyncOptions):
+    """Options controlling a file repository
+    :meth:`~pubtools.pulplib.FileRepository.sync`.
+    """
+
+    remove_missing = pulp_attrib(default=False, type=bool)
+    """If true, as the repository is synchronized, old rpms will be removed.
+    """
+
+    validate = pulp_attrib(default=None, type=bool)
```

Comment:
This seems like another good candidate for omitting.  If you set validate=False, then file corruption is silently ignored... seems odd, why would anyone want to do that?  Can we just force it as True always?